Makefile

```diff
@@ -2,11 +2,12 @@
 all: build-deps build test
 
 ALL_PACKAGES = $(shell go list ./...)
-
-DB_NAME = "todo_db"
+UNIT_TEST_PACKAGES = $(shell go list ./...)
+DB_NAME = "todo_db_dev"
```

Comment:
you do not need the word `db` in the name of DB. It is implicit

Reply:
Done.